x-pack/plugins/canvas/public/apps/export/export/export_app.component.tsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect } from 'react';
+import PropTypes from 'prop-types';
+// @ts-expect-error untyped library
+import Style from 'style-it';
+// @ts-expect-error untyped local
+import { WorkpadPage } from '../../../components/workpad_page';
+import { Link } from '../../../components/link';
+import { CanvasWorkpad } from '../../../../types';
+
+interface Props {
+  workpad: CanvasWorkpad;
+  selectedPageIndex: number;
+  initializeWorkpad: () => void;
+}
+
+export const ExportApp: FC<Props> = ({ workpad, selectedPageIndex, initializeWorkpad }) => {
+  const { id, pages, height, width } = workpad;
+  const activePage = pages[selectedPageIndex];
+  const pageElementCount = activePage.elements.length;
+
+  useEffect(() => initializeWorkpad);
```

Comment:
Looks like this isn't actually calling `initializeWorkpad`, just returning the function? Reporting still looks to work though, so maybe it's not necessary?

Reply:
Or is the `export app` stuff even used? I'm unfamiliar with it.

Reply:
@crob611 It's actually a bug on my part.  That function ends up emitting `fetchAllRenderables`.